sparse_ho/models/lasso.py

```diff
@@ -241,8 +242,8 @@ def _reduce_alpha(alpha, mask):
         return alpha
 
     @staticmethod
-    def _get_jac_t_v(X, y, jac, mask, dense, alphas, v, n_samples):
-        return n_samples * alphas[mask] * np.sign(dense) @ jac
+    def _get_jac_t_v(X, y, jac, mask, dense, alphas, v):
```

Comment:
Honest question: how come this worked before ?

Reply:
We are solving Ax = b  * n_samples, before it solved Ax= b * n_samples , no w Ax / n_samples = b